### PR TITLE
Add `pension_introduction_update` to features.yml

### PR DESCRIPTION
**Pull request by ToddWebDev**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
Add a feature toggle to for pension benefits

## Related issue(s)

- https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/88387

## Testing done

- [x] Tests passing

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by TaiWilkin — DISMISSED** [review]
Looks good

File: config/features.yml [hunk 1]
```diff
@@ -1511,6 +1511,9 @@ features:
     actor_type: user
     description: Implement multiple page response pattern
     enable_in_development: true
+  pension_introduction_update
```
Comment (rmtolmach) [inline config/features.yml:1514]: Your tests are failing because you're missing a `:` at the end of this feature name.

**Comment by rmtolmach** [comment]
@ToddWebDev Please update the PR description

File: config/features.yml (hunk 1 above)
Comment (ToddWebDev) [reply config/features.yml]: Thank you for catching that

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**